addons/website/static/src/js/content/snippets.animation.js

```diff
@@ -1690,6 +1690,22 @@ registry.chart = publicWidget.Widget.extend({
     },
 });
 
+registry.ScrollButton = publicWidget.Widget.extend({
+    selector: '.o_scroll_button',
+    events: {
+        'click': '_onClick',
+    },
+    _onClick: function (ev) {
+        // TODO: Fix the offset when bvr's task is merged (doesn't account for top bars)
+        const offset = this.$el.closest('section').next('section').offset();
+        if (offset) {
+            $('html, body').animate({
+                scrollTop: offset.top,
+            }, 500);
+        }
+    }
+});
```

Comment:
What about extending the `anchorSlide` animation to automatically go to the next section and thus use the same animate without duplicating the code.

Reply:
Good point